### PR TITLE
Create B827EBFFFED61EBD.JSON

### PR DESCRIPTION
**Pull request by yanan522**

### CODE REVIEW

**Comment by yanan522** [comment]
For lora , test

File: B827EBFFFED61EBD [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ 
+“gateway_conf":{
```
Comment (gonzalocasas) [inline B827EBFFFED61EBD]: Invalid double quote character

File: B827EBFFFED61EBD [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{ 
+“gateway_conf":{ 
+"gateway _ID":"B827EBFFFED61EBD", 
+ "servers":[
+ {
+ "server_address":"router.eu.thethings.network", 
+ "serv_port_up"1700,
```
Comment (gonzalocasas) [inline B827EBFFFED61EBD]: Missing colon between key and value.

File: B827EBFFFED61EBD [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{ 
+“gateway_conf":{ 
+"gateway _ID":"B827EBFFFED61EBD", 
+ "servers":[
+ {
+ "server_address":"router.eu.thethings.network", 
+ "serv_port_up"1700, 
+ "serv_port_down"1700,
```
Comment (gonzalocasas) [inline B827EBFFFED61EBD]: Missing colon between key and value.

**Pull request closed without merging**